lib/bb/fetch2/__init__.py

```
@@ -1263,7 +1236,7 @@ def get_checksum_file_list(d):
             found = False
             paths = ud.method.localfile_searchpaths(ud, d)
             for f in paths:
-                pth = ud.decodedurl
+                pth = ud.path
```

Ruff (0.8.2)

Local variable `pth` is assigned to but never used

Remove assignment to unused variable `pth`

(F841)